cves/CVE-2014-3583.yml

```diff
@@ -131,8 +148,11 @@ subsystem:
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
-  answer:
-  name:
+  answer: |
+    Subsystem name found in documentation. The vulnerability is in a module for 
+    auth & authorization. The path name is 'aaa', which also stands for authentication 
+    and authorization.
+  name: Authentication and Authorization
```

Comment:
Why is the vulnerability in this module, was it specific to long auth headers, or did this module control all headers for security purposes? Maybe some more information to include somewhere.

Reply:
Not exactly sure why. It looks like this module handles FastCGI authentication specifically, but it wouldn't explain why (like you mentioned) they wouldn't just handle headers in one uniform place